qutebrowser/utils/log.py

```diff
@@ -505,7 +505,12 @@ def dump_log(self, html=False, level='vdebug'):
             if record.levelno >= minlevel:
                 lines.append(fmt(record))
         return '\n'.join(lines)
-
+        
+    def change_log_capacity(self, capacity):
+        """
+        change log capacity according to user specifcation
+        """
```

Comment:
No need for a docstring for such a short function, the docstring basically says what the function name already says - I think you can just remove it :wink: